apps/trading/components/deposit-container/deposit-container.tsx

```diff
@@ -220,32 +222,43 @@ const DepositForm = ({
         )}
       </FormGroup>
       <FormGroup label="Asset" labelFor="asset">
-        <Controller
-          name="assetId"
-          control={form.control}
-          render={({ field }) => {
-            return (
-              <TradingRichSelect
-                placeholder="Select asset"
-                value={field.value}
-                onValueChange={field.onChange}
-              >
-                {assets.map((a) => {
-                  return (
-                    <TradingRichSelectOption value={a.id} key={a.id}>
-                      <AssetOption asset={a} />
-                    </TradingRichSelectOption>
-                  );
-                })}
-              </TradingRichSelect>
-            );
-          }}
-        />
-        {form.formState.errors.assetId?.message && (
-          <TradingInputError>
-            {form.formState.errors.assetId.message}
-          </TradingInputError>
-        )}
+        <div className="flex flex-col gap-1">
+          <Controller
+            name="assetId"
+            control={form.control}
+            render={({ field }) => {
+              return (
+                <TradingRichSelect
+                  placeholder="Select asset"
+                  value={field.value}
+                  onValueChange={field.onChange}
+                >
+                  {assets.map((a) => {
+                    return (
+                      <TradingRichSelectOption value={a.id} key={a.id}>
+                        <AssetOption asset={a} />
+                      </TradingRichSelectOption>
+                    );
+                  })}
+                </TradingRichSelect>
+              );
+            }}
+          />
+          {asset && !isAssetUSDTArb(asset) && (
+            <Notification
```

Comment:
We are moving away from this component in forms. Can we instead render text similar to TradingInputError but in a warning (orange) color? Perhaps a new TradingInputWarning component, or perhaps TradingInputError could be changed to InputFeedback with a variant prop